iFixit/react-commerce
# Update Chakra to v1.8.3

closes #171 

This PR updates Chakra so that we can make use of up to date frame motion for animations in the upcoming new store home page.

> **Note**: The are also a couple of minor changes that have been attached to this PR as detailed in the changelog below

## Changelog

- Update Chakra and a few related packages
- Refactor code to fix warnings due to updated dependencies
- Mention use of NVM in readme
- Remove default JWT secret from Strapi Admin config: This is to prevent us from relying on a token stored in version control

## QA

1. Visit the Vercel PR preview
2. Visit parts product list page
3. Verify that the product list page still work
